validation-model/src/main/java/org/verapdf/gf/model/impl/external/GFICCProfile.java

```diff
@@ -80,4 +88,9 @@ public Double getversion() {
 	public Boolean getisValid() {
 		return Boolean.valueOf(this.iccProfile.isLooksValid());
 	}
+
+	public boolean isAlternateMatchesIccProfileComponentsNumber(){
```

Comment:
lets rename the method and move log message in it, i.e. make it the "check" method